lib/widgets/details_content.dart

```diff
@@ -0,0 +1,100 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/painting.dart';
+import 'package:marvelapp_flutter/widgets/series_element.dart';
+import '../data/model/response_models/character.dart';
+import '../data/model/response_models/series.dart';
+
+class DetailsContent extends StatelessWidget {
+  final Character? character;
+  final List<Series>? series;
+
+  const DetailsContent(
+      {Key? key, required this.character, required this.series})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    String description = "";
+    if (character?.description != null) {
```

Comment:
I suggest skipping Description UI element if there is no Description resieved from the server.

Reply:
do this with Description and Series. when description equal empty string or series equal empty list it no shows in the screen